src/video/x11/SDL_x11vulkan.c

```diff
@@ -39,6 +39,13 @@
 #define DEFAULT_X11_XCB "libX11-xcb.so.1"
 #endif
 
+SDL_ELF_NOTE_DLOPEN(elfnote_wayland,
```

Comment:
More pasta

Reply:
mmmm, pasta! Spaghetti!

```suggestion
SDL_ELF_NOTE_DLOPEN(elfnote_x11_vulkan,
```